app/src/test/java/com/codesoom/assignment/TaskRepositoryCleaner.java

```diff
@@ -0,0 +1,18 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.TaskRepository;
+
+public class TaskRepositoryCleaner {
```

Comment:
좋습니다! 한편 저는 이렇게도 합니다.

```suggestion
public class TaskRepositoryCleaner implements TestOnly {
```

`TestOnly` 인터페이스는 그냥 비어 있는 인터페이스입니다. 스탬프로 쓰는 거죠.

```
public interface TestOnly {
}
```

이렇게 하면 이 리포지토리 클래스 파일을 열어본 사람이 테스트 전용 클래스라는 것을 바로 알 수 있겠죠.